frontend/packages/console-app/src/components/quick-starts/controller/QuickStartTaskHeader.tsx

```diff
@@ -71,12 +71,13 @@ const QuickStartTaskHeader: React.FC<QuickStartTaskHeaderProps> = ({
       <Title headingLevel="h3" size={size} className={classNames}>
         <TaskIcon taskIndex={taskIndex} taskStatus={taskStatus} isActiveTask={isActiveTask} />
         {title}
+        {isActiveTask && subtitle && (
+          <Text component={TextVariants.a} className="text-secondary">
+            {' '}
+            {subtitle}
+          </Text>
```

Comment:
You created a class name never used it. Also you need to remove the space from the inline-block element.
```suggestion
          <>
            {' '}
            <small className="co-quick-start-task-header__subtitle text-secondary">
              {subtitle}
            </small>
          </>
```

Reply:
updated